### ISSUE TITLE
Aeon z-wave stick on raspberry pi

### ISSUE THREAD

**Issue by omegacore** [openhab/openhab#2350]
I am having trouble with the aeon z-wave stick on the raspberry pi 2. It shows up under /dev as ttyUSB0. But my binding cannot find the device while loading. What am I missing? Using raspbian.

**Comment by pierok13** [comment]
you may face an authorization issue to your /dev/ttyUSB0 or you may follow this guide as I did (i'm also working with a RPI2 and aeon labs z-Stick)

https://github.com/openhab/openhab/wiki/Samples-Tricks#how-to-configure-openhab-to-connect-to-device-symlinks-on-linux

**Comment by omegacore** [comment]
Thanks pierok. I'll try this when I get home today. Did something change with openhab? I didn't need to do this before, granted I was running openhab on my ubuntu desktop...

**Comment by B34N** [comment]
Did you find a solution? I'm having the same issue on a new rpi 2 with ahte Aeon stick. It's as if openhab doesn't see the stick at all and it's on /dev/ttyUSB0. Everything was setup under default user pi but open hab is run through sudo

**Comment by bdickenson** [comment]
Was just browsing the error stream and saw this one....I have an Aeon zstick 2 and RPi 2 running basically OOB.   I had no issues (other than the usual "pilot error" type)  on the porting OH from Windows host to Pi 2 using the same zstick other than port naming (my showed as /dev/ttyUSB0)

**Comment by SkynetHome** [comment]
I have been running the same Aeon Stick on a Raspberry Pi 2 for over a month now with no problems. The only time I have had an issue with the /dev/tty/USBO was when I unplugged the stick while the pi was running to take it to the other side of the house and manually include a device and then once plugged back in I got an error. A simple restart of the Pi 2 solved this and so anytime I need to manually include with the z stick I shutdown the Pi first, remove the stick and include, plug back in and reboot and all is well!
